app/cmd/controller.go

```diff
@@ -104,6 +109,11 @@ func startController(c *cli.Context) error {
 		return err
 	}
 
+	engineReplicaTimeout := c.Int64("engine-replica-timeout")
+
+	engineReplicaTimeout = controller.DetermineEngineReplicaTimeout(engineReplicaTimeout)
+	iscsiTargetRequestTimeout := controller.DetermineIscsiTargetRequestTimeout(engineReplicaTimeout)
```

Comment:
All related, so can put together w/o empty line.

```go
engineReplicaTimeout := c.Int64("engine-replica-timeout")
engineReplicaTimeout = controller.DetermineEngineReplicaTimeout(engineReplicaTimeout)
iscsiTargetRequestTimeout := controller.DetermineIscsiTargetRequestTimeout(engineReplicaTimeout)

```

Reply:
Addressed. Thank you.